src/practice4/task3_MessageBus/entity/BusMessage.java

```diff
@@ -0,0 +1,49 @@
+package practice4.task3_MessageBus.entity;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * This class contains information about the bus: bus number, date and time of departure.
+ *
+ * @author Kiryl_Vinakurau
+ */
+public class BusMessage {
```

Comment:
"Message bus" is just a pattern name

Reply:
Name of class BusMesage changed to TransportTimetable